.gitignore

```diff
@@ -4,3 +4,4 @@ erl_crash.dump
 *.ez
 /doc
 .DS_Store
+.elixir_ls/
```

Comment:
This doesn't apply specifically to the repository and probably should not be included.

It's an unrelated request, but if you wanted to remove `.DS_Store`, that'd be cool too, but not necessary 🙂

Platform/tooling specific stuff typically ends up using git global ignore configuration: https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
Done.